### ISSUE TITLE
Open for all: Design the New Tab page

### ISSUE THREAD

**Issue by rag-hav** [opencodeiiita/Contest-Countdown#49]
Currently the new tab page looks like this

![image](https://user-images.githubusercontent.com/60528983/136540328-2bb7e7d0-6c92-4cf2-aa3f-92047dfa3b8d.png)

And, this is the normal new tab page

![Screenshot from 2021-10-08 15-52-42](https://user-images.githubusercontent.com/60528983/136540496-72f09479-9ee2-4e65-9d50-b80883a82142.png)

Design a new tab page that combines these two! It should somewhat (or better) look like the new tab page of the default chrome while also elegantly displaying the timer and contest information. Do not be shy to propose extra buttons or drop down lists, or to make the contest details hidden by default. Also feel free to use icons from chrome/assets/icons

Bottom line is to design something aesthetically pleasing! Let your imagination run wild.

Since this is a designing issue, and design is subjective. The final decision to accept or reject a PR rests with the mentor.

Selected design will get bonus 20 points.
